Source/DafnyLanguageServer.Test/_plugins/PluginsQuickFixTest.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.Dafny;
+using Microsoft.Dafny.LanguageServer.Plugins;
+using System.Collections;
+using Microsoft.Boogie;
+using Microsoft.Dafny.LanguageServer.Language;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace PluginsQuickFixTest {
+  /// <summary>
+  ///  Small plugin that detects all extern methods and verifies that there are test methods that actually invoke them.
```

Comment:
Where is the code that detects extern methods?

Reply:
> Where is the code that detects extern methods?

The comment was oudated. Updated.